npc/re/merchants/eden_market.txt

```diff
@@ -97,7 +96,21 @@ OnClock0000:
 	npcshopupdate "para_hei10",23016,0,1;
 	npcshopupdate "para_key10",7026,10;
 	npcshopupdate "para_key10",7027,10;
-	npcshopupdate "para_wp00",1181,0,1;
+	npcshopupdate "para_wp00",1181,0,1;	// 2h sword - Tae Goo Lyeon
+	npcshopupdate "para_wp00",1231,0,1;	// MATK dagger - Bazerald 
+	npcshopupdate "para_wp00",13039,0,1;	// ATK dagger - Ivory Knife
+	npcshopupdate "para_wp00",1140,0,1;	// 1h sword - Byeollungum
+	npcshopupdate "para_wp00",1142,0,1;	// 1h spear - Hunting Spear
+	npcshopupdate "para_wp00",1478,0,1;	// 2h spear - Ahlspiess - 2h spear is a tough one, need peer review
+	npcshopupdate "para_wp00",1306,0,1;	// War Axe - 1h axes suck lol
+	npcshopupdate "para_wp00",1137,0,1;	// 2h axe - Hurricane's Fury, other optioon would be Veteran Axe 1384
```

Comment:
Currently [Excalibur](https://www.divine-pride.net/database/item/1137), I think [Hurricane's Fury](https://www.divine-pride.net/database/item/1377/hurricane-fury-1) is a nice option over Veteran Axe